test/accessibility/reporter/Report.html

```diff
@@ -0,0 +1,215 @@
+        <div class="header">
+            <div class="titleText">
+                <div>
+                    <i class="fas fa-universal-access" style="font-size:60px;color:white;"> </i>
+                </div>
+                
+                <div>
+                    <div id="reportHeaderTitle">Accessibility Test Report for Expert UI</div>
+                    <div id="reportHeaderAppName">{{data.appName}}</div>
+                </div>
+               
+            </div>
+           
+            <div class="stats">
+                <i class="fas fa-check-circle" style="font-size:40px;color:lightgreen;"></i>{{data.passed}}
+              <i class="fas fa-times-circle" style="font-size:40px;color:red;"></i>{{data.failed}}  
+            </div>
+            
+        </div>
+    
+
+    <div class="testResult" ng-repeat="test in data.tests">
+       <div class="testTitle" ng-class="{pass : test.status == 'passed', fail: test.status == 'failed'}"  ng-click="showDetails(test)">{{ test.name }}</div>
+       <div ng-if="test.status == 'failed'">
+           <pre>
+              <code>{{test.error}}</code> 
+           </pre>
+       </div>
+       
+       <div class="a11yResult" ng-class="{show: test.show, hide:!test.show}">
+            <!-- {{test.a11yResult}} -->
+            <table class="resultTable">
+                <tr>
+                    <th>Document title:</th>
+                    <td>{{test.a11yResult.documentTitle}}</td>
+                </tr>
+                <tr>
+                    <th>Page URL:</th>
+                    <td>{{test.a11yResult.pageUrl}}</td>
+                </tr>
+                <tr>
+                    <th>Total issues:</th>
+                    <td>{{test.a11yResult.issues.length}}</td>
+                </tr>
+            </table>
+        
+            <h4>Issues:</h4>
+            <div ng-repeat="issue in test.a11yResult.issues">
+                <table class="issuesTable">
+                    <tr> <th>Issue code</th><td>{{issue.code}}</td></tr>
+                     <tr> <th>Issue type</th><td>{{issue.type}}</td></tr>
+                    <tr>
+                        <th>Runner</th>
+                        <td>{{issue.runner}}</td>
+                    </tr>
+                    <tr><th>Message</th> <td>{{issue.message}}</td></tr>
+                    <tr> <th>Selector</th><td>{{issue.selector}}</td></tr>
+                    <tr>
+                        <th>Context</th>
+                        <td><pre><code>{{issue.context}}</code></pre></td>
+                    </tr>
+                    
+                </table>
+             
+            </div>
+            <img ng-src="{{test.a11yResult.screenshot}}" alt="a11y test page" width="500" height="500">
+
+        </div>
+    </div>
+
+    
+    </div>
+        <script src="./report_output.js" ></script>
+
+        <script>
+            var data = replacejsoncontent;
+
+            var app = angular.module('myApp', []);
```

Comment:
any particular reason we're using angular v1?

Reply:
@paddybasi no specific reason.this is custom HTML report we created. We using for Model view control only with angularJS in html file.